core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -40,24 +40,36 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
               <span ng-if="getAverageRating()">
                 <[getAverageRating() | number:1]>
               </span>
-              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A">
+              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A" class="inactive-summary-tile-attribute ">
```

Comment:
Here and below:
- add 'oppia-' as a prefix to the CSS class. We do this for classes introduced within Oppia to distinguish them from classes due to other third-party libs.
- Remove the unnecessary space at the end.